File: tests/expected/comment_unsupported.kt
fun do_unsupported() {
    val a = 1
// dict comprehension ((key.toInt() + 1), (value.toInt() + 1)) unimplemented on line 9:4
    val b = (a != 0)
    if (true) {
        val __tmp1 = if (b) "True" else "False"
        println("$__tmp1")
    }
}

fun main(argv: Array<String>) {
    do_unsupported()
}

File: tests/expected/comment_unsupported.dart
// @dart=3.4
import 'package:sprintf/sprintf.dart';

do_unsupported() {
  final int a = 1;
/* dict comprehension ((key + 1), (value + 1)) unimplemented on line 9:4 */;
  final bool b = (a != 0);
  print(sprintf("%s", [b ? ("True") : ("False")]));
}

main(List<String> argv) {
  do_unsupported();
}
